lib/matplotlib/testing/conftest.py

```diff
@@ -45,6 +45,30 @@ def mpl_test_settings(request):
                 'skip_on_importerror', False)
             prev_backend = matplotlib.get_backend()
 
+            # special case Qt backend importing to avoid conflicts
+            if backend == 'Qt4Agg':
```

Comment:
.lower().startswith("qt4") to be robust against lowercase and also work with qtcairo?  (same below for qt5)